tensorflow/python/kernel_tests/pooling_ops_test.py

```diff
@@ -947,7 +947,7 @@ def _ConstructAndTestGradient(self,
           output_sizes,
           x_init_value=x_init_value,
           delta=1e-2)
-    print("%s gradient error = " % func_name, err)
+    tf_logging.info("%s gradient error = " % func_name, err)
```

Comment:
This is broken, should be:
 `tf_logging.info("%s gradient error = %.4f" % (func_name, err))`